operator_ui/src/pages/Jobs/transformJobRuns.ts

```diff
@@ -61,13 +61,16 @@ export const transformPipelineJobRun = (jobSpecId: string) => (
   const stratify = parseDot(
     `digraph {${jobRun.attributes.pipelineSpec.dotDagSource}}`,
   )
-
+  let taskRuns: PipelineTaskRun[] = []
+  if (jobRun.attributes.taskRuns != null) {
```

Comment:
Could the BE always return an array instead of null? That way you wouldn't need to do this check here.

Reply:
its actually messier to do on the backend - you have to allocate an empty slice in a bunch of places - seems cleanest to do it here